src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -569,8 +573,13 @@ public void setEnableScheduledScanJobs(boolean enabled) {
                     " availble prior to Android 5.0");
             return;
         }
+        if (enabled && android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Quick question: If this is concerned about logging a warning when _disabling_ scan jobs, shouldn't this condition be inverted, and check for `!enabled` instead?

Reply:
Good catch, thanks!  It should indeed be !enabled.  Care to open an issue (or for bonus points a PR)?

Reply:
Sure! Raised as #700